js_modules/dagster-ui/packages/ui-core/src/ticks/SensorDryRunDialog.tsx

```diff
@@ -195,6 +213,35 @@ const SensorDryRun = ({repoAddress, name, currentCursor, onClose, jobName}: Prop
     setLaunching(true);
 
     try {
+      if (dynamicPartitionRequests?.length) {
+        await Promise.all(dynamicPartitionRequests.map(async (request) => {
+          if (request.type === DynamicPartitionsRequestType.ADD_PARTITIONS) {
+            await Promise.all((request.partitionKeys || []).map(async (partitionKey) => {
+              await createPartition({
+                variables: {
+                  repositorySelector: {
+                    repositoryName: repoAddress.name,
+                    repositoryLocationName: repoAddress.location,
+                  },
+                  partitionsDefName: request.partitionsDefName,
+                  partitionKey,
+                },
+              });
+            }));
+          } else if (request.partitionKeys && request.partitionKeys.length) {
+            await deletePartition({
+              variables: {
+                repositorySelector: {
+                  repositoryName: repoAddress.name,
+                  repositoryLocationName: repoAddress.location,
+                },
+                partitionsDefName: request.partitionsDefName,
+                partitionKeys: request.partitionKeys,
+              },
+            });
+          }
+        }));
+      }
       if (executionParamsList) {
         await launchMultipleRunsWithTelemetry({executionParamsList}, 'toast');
         onCommitTickResult(); // persist tick
```

Comment:
Should this be inside of the `if (executionParamsList) {` ?

Also would be nice if we can do all of this atomically on the server. Especially comiting the tick.

Reply:
I don't think so... the execution params are associated with run requests, and the dynamic partition results are associated with dynamic partition requests.  They're orthogonal.

The point stands though, that maybe this should be a different button?  or we should change the button text at least to apply whatever changes can be made?

So maybe we make the button clickable even if there are no run requests (and thus `canLaunchAll` is falsey).  And maybe we can rename the button to say something like:
```Apply requests and save tick```